cli/node/interfaces.py

```diff
@@ -19,3 +19,9 @@ def get_peers(self):
         Get the node's peers.
         """
         pass
+
+    def get_info(self):
+        """
+        Get node information.
```

Comment:
Perhaps here needs more information about what it is, for greater clarity.
What do you think about that @anastasiia-bilova?

Reply:
@Alladin9393, done.